.github/workflows/build.yml

```diff
@@ -0,0 +1,73 @@
+name: Build
+
+on:
```

Comment:
We could configure the repository to have "main"/"develop" branches, and require that PRs go into the develop branch. And later, we could also split the "build-image" and "build-library" into separate workflows, and then run "build-image" only when needed.

Reply:
* I just created a `develop` branch, that is now the default branch. This PR now targets the `develop` branch.
* For the split of the CICD, I would do it in a followup :wink: